npm.el

```diff
@@ -22,41 +22,47 @@
 
 ;;; Code:
 
-(require 'cl)
+(eval-when-compile
+  (require 'cl))
```

Comment:
Indeed, actually I removed this `require` at first and did not see any broken functionality.  I'll check further then remove it if it is not needed.